Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -1465,6 +1469,10 @@ public BaseItemDto Map(BaseItemEntity entity, BaseItemDto dto)
         {
             dto.ImageInfos = entity.Images.Select(Map).ToArray();
         }
+        else
+        {
+            System.Console.WriteLine();
+        }
```

Comment:
Forgot to remove this?

Reply:
no ... i just wanted to see if you are actually reviewing ;-)